src/panel/direction/DirectionPanel.jsx

```diff
@@ -323,7 +325,7 @@ export default class DirectionPanel extends React.Component {
             />}
           </div>
           {!activePreviewRoute && origin && destination &&
-            <Panel resizable marginTop={160} >
+            <Panel resizable marginTop={this.mobileFormRef.current.offsetHeight} >
```

Comment:
Great :+1:  I think it's safer to check for mobileFormRef.current before.
![image](https://user-images.githubusercontent.com/2981774/95876206-1d5d8380-0d73-11eb-9174-90ea310df83a.png)